src/main/java/de/tum/in/www1/artemis/web/rest/dto/LectureUnitSplitDTO.java

```diff
@@ -0,0 +1,9 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+import java.time.ZonedDateTime;
+
+/**
+ * Information of a lecture unit to be splitted
+ */
+public record LectureUnitSplitDTO(String unitName, ZonedDateTime releaseDate, Integer startPage, Integer endPage) {
```

Comment:
```suggestion
public record LectureUnitSplitDTO(String unitName, ZonedDateTime releaseDate, int startPage, int endPage) {
```
Can those be primitive types as well, or are there lecture unit splits where no start/end is given (e.g. starting implicitly at the first slide, ending at the last slide)?

Reply:
Actually yes. Users will have to give the start/end pages even if there is one unit  (e.g. starting implicitly at the first slide, ending at the last slide).